presto-parser/src/main/java/com/facebook/presto/sql/ExpressionFormatter.java

```diff
@@ -566,6 +573,53 @@ static String formatSortItems(List<SortItem> sortItems, boolean unmangleNames)
                 .iterator());
     }
 
+    static String formatGroupBy(List<GroupingElement> groupingElements)
+    {
+        ImmutableList.Builder<String> resultStrings = ImmutableList.builder();
+
+        for (GroupingElement groupingElement : groupingElements) {
```

Comment:
That will require me to have getColumns() return a list of Expressions. Should I make that change?